dnf-repoconfig-daemon/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
I'd rather set the standard just for the daemon target(s):
```set_property(TARGET tgt PROPERTY CXX_STANDARD 17)```
see: https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html